torch/nn/quantizable/modules/rnn.py

```diff
@@ -0,0 +1,288 @@
+import numbers
+from typing import Optional, Tuple
+import warnings
+
+import torch
+from torch import Tensor
+
+"""
+We will recreate all the RNN modules as we require the modules to be decomposed
+into its building blocks to be able to observe.
+"""
+
+class LSTMCell(torch.nn.Module):
```

Comment:
for all of these, can we add a `_get_name` method which includes the information that this is not the vanilla `nn.LSTMCell`, etc